### PR TITLE
Added a form in index.mdx

### PR DESCRIPTION
**Pull request by Deepak-Choudhary0**
**Description**

This PR fixes #

This PR solves the issue #4219 
by adding a form at https://layer5.io/learn/service-mesh-books/the-enterprise-path-to-service-mesh-architectures-2nd-edition.
Still need a review before making the merge, like the data shown after signing up for book. I'm open for any improvements.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/service-mesh-books/the-enterprise-path-to-service-mesh-architectures-2nd-Edition/index.mdx [hunk 1]
```diff
@@ -14,11 +14,23 @@ redirect_from:
...
+ */
+   <Col lg={5} md={6} sm={12} style={{ padding: "0" }}>
+      <CommonForm
+        title="Get the Book"
+        form="The Enterprise path to service mesh Architectures 2nd Edition"
+        account_desc="Send the The Enterprise path to service mesh Architectures 2nd Edition book in the Mail."
+        submit_title="You will get book soon in the mail!"
+        submit_body="Please patiently await for The Enterprise path to service mesh Architectures 2nd Edition Book."
+      />
+    </Col>
```
Comment (leecalcote) [inline src/collections/service-mesh-books/the-enterprise-path-to-service-mesh-architectures-2nd-Edition/index.mdx:33]:
```suggestion
   <Col lg={5} md={6} sm={12} style={{ padding: "0" }}>
      <CommonForm
        title="Get the Book"
        form="epsma-book"
        account_desc="Email the The Enterprise Path to Service Mesh Architectures 2nd Edition"
        submit_title="Your copy of the book will arrive in your mailbox shortly."
        submit_body="While waiting for your copy to arrive take a free learning path - https://layer5.io/learn/learning-path"
      />
    </Col>
```

**Comment by leecalcote** [comment]
Thanks, @Deepak-Choudhary0

**Comment by l5io** [comment]
🚀 Preview for commit 0d00df7f9698d6a1872b656f4e6d4757d81364cc at: https://646dfec62d81770c35b5f18d--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 168e038178e6bc605a1fc7d627c5e27500b0be4f at: https://646e1a819244ec05242d4f60--layer5.netlify.app

**Comment by leecalcote** [comment]
> > Thanks, @Deepak-Choudhary0
> 
> Hey lee any update on link? like even if you click on image of the book it leave you at another link. So can i remove that link completely?

@Deepak-Choudhary0, yes, you can remove the external link completely. 👍

**Comment by Deepak-Choudhary0** [comment]
> > > Thanks, @Deepak-Choudhary0
> > 
> > 
> > Hey lee any update on link? like even if you click on image of the book it leave you at another link. So can i remove that link completely?
> 
> @Deepak-Choudhary0, yes, you can remove the external link completely. 👍

Done any other review??

**Comment by leecalcote** [comment]
Please make this a clickable `<Link>` --> https://layer5.io/learn/learning-path

**Comment by leecalcote** [comment]
The form action isn't invoking the necessary webhook to handle submission on the server-side.

**Comment by l5io** [comment]
🚀 Preview for commit 8276b4475bcca51e7adc8cab6feb278d75da5ba6 at: https://646eefeaedf95673549e8c14--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 4a03c2f66df7b86d1df0d504d2747e9aa5e79994 at: https://646f42f90283630986d6fe1b--layer5.netlify.app

**Comment by Deepak-Choudhary0** [comment]
> Please make this a clickable `<Link>` --> https://layer5.io/learn/learning-path

snap

**Comment by Savio629** [comment]
@Deepak-Choudhary0 Let's discuss this on websites call, add this as an agenda in [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) if you would. 
The meeting is at 5.30 pm IST.

**Review by theBeginner86 — COMMENTED** [review]
@Deepak-Choudhary0 

Would you please fix this styling issues?


On submission:

Screenshot 2023-06-01 at 4 57 24 PM

**Comment by theBeginner86** [comment]
I've handled all the server-side changes. // @Deepak-Choudhary0

**Comment by theBeginner86** [comment]
Also switching CTA from Docker Extension to Playground or MeshMap might be worth it. 🤔

**Comment by Deepak-Choudhary0** [comment]
> @Deepak-Choudhary0
> 
> Would you please fix this styling issues?
> 
> On submission:
> 
> <img alt="Screenshot 2023-06-01 at 4 57 24 PM" width="1438" src="https://user-images.githubusercontent.com/73700530/242582886-fa3d84ae-09b8-4302-bb9c-39c3893d394f

Yeah sure, that's where i am working.

**Comment by Savio629** [comment]
@Deepak-Choudhary0 Updates?

**Comment by Deepak-Choudhary0** [comment]
I will show it on Monday meetings as i have done the output(the confirmation message) part styling , just the input form needs to be corrected too.

**Comment by Ghat0tkach** [comment]
> @Deepak-Choudhary0
> 
> Would you please fix this styling issues?
> 
> On submission:
> 
> Screenshot 2023-06-01 at 4 57 24 PM

Well this could be resolved by changing ```.book class``` 
```float:none```  from  ```float:right```  when we click on submit button.
What are your thoughts @Savio629 @Shivam-AfA @Deepak-Choudhary0 


https://github.com/layer5io/layer5/assets/59855919/b9b02e19-c5b4-4424-ae09-bc523de7d5e7

**Comment by vishalvivekm** [comment]
> I will show it on Monday meetings as i have done the output(the confirmation message) part styling , just the input form needs to be corrected too.

Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) @Deepak-Choudhary0

**Comment by Deepak-Choudhary0** [comment]
> > @Deepak-Choudhary0
> > Would you please fix this styling issues?
> > On submission:
> > Screenshot 2023-06-01 at 4 57 24 PM
> 
> Well this could be resolved by changing `.book class` `float:none` from `float:right` when we click on submit button. What are your thoughts @Savio629 @Shivam-AfA @Deepak-Choudhary0
> 

>  Recording.2023-06-11.003858.mp4

This will lead the input form to be bigger 
Screenshot from 2023-06-12 16 02 09

**Comment by Savio629** [comment]
@Deepak-Choudhary0 updates?

**Comment by Deepak-Choudhary0** [comment]
> > > > @Deepak-Choudhary0
> > > > Would you please fix this styling issues?
> > > > On submission:
> > > > Screenshot 2023-06-01 at 4 57 24 PM
> > > 
> > > 
> > > Well this could be resolved by changing `.book class` `float:none` from `float:right` when we click on submit button. What are your thoughts @Savio629 @Shivam-AfA @Deepak-Choudhary0
> > 
> > 
> > > Recording.2023-06-11.003858.mp4
> > 
> > 
> > This will lead the input form to be bigger
> > Screenshot from 2023-06-12 16 02 09
> 
> Im saying after when we click submit button , because the issue arises when we submit the form.

hey @Ghat0tkach do you have anything in mind how we can do that float:none when user clicks on submit , mine mind is blank.

**Comment by Ghat0tkach** [comment]
@Deepak-Choudhary0 ,  Mine approach is simple  , we have to use two functions one for submitting email to backend and another is to making float to none.


We can use
```
function floatRight() {
  document.getElementById("myImg").style.cssFloat = "right";
}

function floatLeft() {
  document.getElementById("myImg").style.cssFloat = "left";
}
```

**Comment by Deepak-Choudhary0** [comment]
@Ghat0tkach how will we trigger this, because in commonform component there is no prop to handle like onclick,onsuccess to trigger this function

**Comment by Savio629** [comment]
@Deepak-Choudhary0 Maybe you can check if "thank-you-box" class is rendering/visible and then append the `float:none` to the "book" class.

**Comment by Deepak-Choudhary0** [comment]
yeah @Savio629  but i am not getting how will i implement this function such that when "ThankYouBox" renders after user successfully registers for book email , the "Book" class should be made float:none

**Comment by vishalvivekm** [comment]
> yeah @Savio629 but i am not getting how will i implement this function such that when "ThankYouBox" renders after user successfully registers for book email , the "Book" class should be made float:none

Let's discuss this on websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) if you would :)

**Comment by Savio629** [comment]
@Deepak-Choudhary0 Did you figure it out?

**Comment by Deepak-Choudhary0** [comment]
let @Ghat0tkach submit a pr then w'll make a feedback

**Pull request closed without merging**